recipes/scheil/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "scheil" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "de43e05fb9fdfb084fa3c7709000cc81c88292eaee24a0ac020c52c032c0c809" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+  skip: true  # [py<36]
```

Comment:
Please make this `noarch: python`.   It really reduces the number of artifacts in the system